pkg/apis/batch/types.go

```diff
@@ -33,6 +33,9 @@ import (
 // always track jobs using finalizers.
 const JobTrackingFinalizer = "batch.kubernetes.io/job-tracking"
 
+// We will use batch.kubernetes.io/ as a prefix for all labels
+const JobLabelPrefix = "batch.kubernetes.io/"
```

Comment:
Nit: `BatchLabelPrefix` seems more appropriate

Reply:
"batch" is the pkg name, so maybe just "LabelPrefix" ?

Reply:
Updated everywhere. Good call on the fact that this could be a constant.